.travis.yml

```diff
@@ -1,13 +1,33 @@
 sudo: false
+dist: trusty
 language: node_js
 node_js:
   - 'node'
-  - '4.2'
-  - '0.12'
-  - '0.10'
+  - 'lts/argon'
 
 before_script:
-
-  # Set up a virtual screen for Firefox.
+  # Check if the current version is equal to the major version for the env.
+  - 'export IS_INSTALLED="$(npm list video.js | grep "video.js@$VJS")"'
+  # We have to add semicolons to the end of each line in the if as Travis runs
+  # this all on one line.
+  - 'if [ -z "$IS_INSTALLED" ]; then
+      echo "INSTALLING video.js@>=$VJS.0.0-RC.0 <$(($VJS+1)).0.0";
```

Comment:
I didn't investigate as to why they changed their default plugin travis build settings. It might be removable. 

I do recommend you split off a videojs-5x branch before merging this.